modes/lisp-mode/lisp-mode.lisp

```diff
@@ -140,7 +140,9 @@
 (defun buffer-package (buffer &optional default)
   (let ((package-name (buffer-value buffer "package")))
     (if package-name
```

Comment:
This if might not needed anymore.

Reply:
In my system, Lem still dies when opening file "iterate.lisp".

Reply:
> This if might not needed anymore.

Excuse-me, I didn't understand what might not be needed. Do you mean the test `(if package-name [...])`?

Reply:
Well, removed that test and it seems to work.

Reply:
Another problem is with `*lisp-repl*` buffer in Lisp mode: the package is set to "NIL". Made one more change so that doesn't happen anymore.